bin/git-commands.js

```diff
@@ -86,6 +86,44 @@ async function pushTags() {
   return output
 }
 
+async function checkout(branchName) {
+  const stdout = await execAsPromise(`git checkout ${branchName}`)
```

Comment:
so are these checkouts working because the repos are public?

Reply:
For CI I'd assume so as I believe the GitHub user only has immediate repo access but I didn't dig into it.